stacks/openfaas/deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -e
+
+kubectl apply -f https://raw.githubusercontent.com/openfaas/faas-netes/master/namespaces.yml
```

Comment:
Mind including the namespace manifest inline? ie:

```
cat <<EOF | kubectl apply -f -
apiVersion: v1
kind: Namespace
metadata:
  name: openfaas
  labels:
    role: openfaas-system
    access: openfaas-system
    istio-injection: enabled
---
apiVersion: v1
kind: Namespace
metadata:
  name: openfaas-fn
  labels:
    istio-injection: enabled
    role: openfaas-fn
EOF
```

Reply:
I would rather track it from the main repo because it would better match the helm install instructions. Can that work?

Reply:
If it's not workable, let me know and I can hard-code the namespace into the shell scripts.

Reply:
Given that it matches the Helm install instructions, the current implementation looks good to me.